test/e2e/main_test.go

```diff
@@ -45,17 +47,20 @@ func init() {
 	flag.StringVar(&taintKey, "taint-key", defaultTaintKey, "the key of the taint that is expected to be associated with the virtual-kubelet node to test")
 	flag.StringVar(&taintValue, "taint-value", defaultTaintValue, "the value of the taint that is expected to be associated with the virtual-kubelet node to test")
 	flag.StringVar(&taintEffect, "taint-effect", defaultTaintEffect, "the effect of the taint that is expected to be associated with the virtual-kubelet node to test")
-	flag.Parse()
```

Comment:
Running flag.Parse in init causes issues because other packages check for Parsed() before they add their flags (in their inits)